flink-connector-debezium/src/main/java/com/alibaba/ververica/cdc/debezium/table/RowDataDebeziumDeserializeSchema.java

```diff
@@ -238,7 +238,7 @@ private long convertToLong(Object dbzObj, Schema schema) {
 
     private double convertToDouble(Object dbzObj, Schema schema) {
         if (dbzObj instanceof Float) {
-            return (double) dbzObj;
+            return ((Float) dbzObj).doubleValue();
```

Comment:
could we add a test for this, this is pretty interesting, you may meet precision problem.